### PR TITLE
Add disableremoteplayback attribute to VideoHTMLAttributes interface

### PR DESCRIPTION
**Pull request by apetta**
## Changes

Addition of `disableremoteplayback` to the `VideoHTMLAttributes` interface for TypeScript support.
No code changes needed as it's already handled by the regex in `packages/astro/src/runtime/server/render/util.ts`

## Testing

N/A - Type change only

## Docs

N/A - Type change only

### CODE REVIEW

File: packages/astro/astro-jsx.d.ts [hunk 1]
```diff
@@ -1057,6 +1057,7 @@ declare namespace astroHTML.JSX {
 		poster?: string | undefined | null;
 		width?: number | string | undefined | null;
 		disablepictureinpicture?: boolean | string | undefined | null;
+		disableremoteplayback?: boolean | string | undefined | null;
```
Comment (Princesseuh) [inline packages/astro/astro-jsx.d.ts:1060]: I think this should be on the media interface instead, as audio can also use this

**Comment by Princesseuh** [comment]
Do you have a reference for this attribute existing? I can only find reference to the `disableRemotePlayback` property on JavaScript HTMLMediaElement, but not to the corresponding HTML attribute.

**Comment by apetta** [comment]
> Do you have a reference for this attribute existing? I can only find reference to the `disableRemotePlayback` property on JavaScript HTMLMediaElement, but not to the corresponding HTML attribute.

Yes here's one from [mdn](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/disableRemotePlayback).

I use it fairly often to get rid of the cast/chromecast button on videos.

**Comment by apetta** [comment]
@Princesseuh You're right it's only a JS property. 

Is there a better way to handle the typing here given that?

**Comment by Princesseuh** [comment]
Closing since it isn't an attribute, feel free to re-open if it does become one, though!

**Comment by apetta** [comment]
@Princesseuh Actually, I found a [reference](https://www.w3.org/TR/remote-playback/#the-disableremoteplayback-attribute). It's in draft _but stable_ status as of March 2024.

**Comment by Princesseuh** [comment]
> @Princesseuh Actually, I found a [reference](https://www.w3.org/TR/remote-playback/#the-disableremoteplayback-attribute). It's in draft _but stable_ status as of March 2024.
> 
> Should I re-open? Seems like a harmless but useful addition given the stability

Ah, interesting! It's weird that it's `disableRemotePlayback` and not `disableremoteplayback`, HTML attributes are typically not in camel case 🤔 That works for me though, at least we have a document to refer to!

**Review by Princesseuh — APPROVED** [review]
Thank you! A rare attribute, but it apparently does exists!

**Comment by apetta** [comment]
> > @Princesseuh Actually, I found a [reference](https://www.w3.org/TR/remote-playback/#the-disableremoteplayback-attribute). It's in draft _but stable_ status as of March 2024.
> > Should I re-open? Seems like a harmless but useful addition given the stability
> 
> Ah, interesting! It's weird that it's `disableRemotePlayback` and not `disableremoteplayback`, HTML attributes are typically not in camel case 🤔 That works for me though, at least we have a document to refer to!

Awesome! I believe html attributes are not case sensitive, but yes a strange choice

**Pull request merged**